html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get"></form>
+  <div>
+    <label for="email">Email</label>
+    <br>
+    <input type="text" name="email">
+    <br>
+    <label for="why">Why is coding cool?</label>
+    <br>
+    <select name="why">
+      <option value="opion">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+  </div>
+  <div>
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" name="language" value="HTML" /> HTML
```

Comment:
Make sure to have a label for each of these inputs. You don't want text content floating in the nether